src/Components/Elements/DragAndDrop.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  closestCenter,
+  DndContext,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+} from "@dnd-kit/core";
+import {
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import SortableElement from "./SortableElement";
+
+export default function DragAndDrag(props) {
```

Comment:
This was meant to be a helper for reducing some of the boilerplate code (not boilerplate as in what this app deals with) needed. This isn't currently being used, but would ideally be used in the sections list in the GrantsShow / GrantsFinalizeShow